Plugins/IntelliJ/src/main/resources/META-INF/plugin.xml

```diff
@@ -1,4 +1,4 @@
-<idea-plugin>
+<idea-plugin require-restart="false">
```

Comment:
oh nice. when is this required vs not?

Reply:
During the analysis of the plugin, Jetbrains included this in the report:

> **Plugin can probably be enabled or disabled without IDE restart**
IDE restart may be unnecessary to enable or disable Android Testify - Screenshot Instrumentation Tests 0.0.1-alpha because the plugin satisfies the requirements for dynamic plugins. Yet the unloading may be restricted if the plugin's class loader cannot be unloaded dynamically or if the plugin has complex dependencies. Please check manually that the IDE allows to install and uninstall Android Testify - Screenshot Instrumentation Tests 0.0.1-alpha without restart.

It seems like this plugin meets the requirements and doesn't do anything that would prevent a hot-reload.